CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to zopefoundation projects
```

Comment:
Does `zopefoundation` need to be capitalized?

Reply:
No, that is the correct name for the GitHub organization